Moodis/Moodis/cameraForm.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AForge.Video;
+using AForge.Video.DirectShow;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace moodie
```

Comment:
moodis not moodie. This is also going to edit a lot of references so be careful when renaming.